EIPS/eip-2583.md

```diff
@@ -0,0 +1,295 @@
+---
+eip: 2583
+title: Penalty for account trie misses
+author: Martin Holst Swende (@holiman)
+discussions-to: https://ethereum-magicians.org/t/eip-2583-penalties-for-trie-misses/4190
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-02-21
+---
+
+
+## Simple Summary
+
+This EIP propose to introduce a gas penalty for opcodes which access the account for trie non-existent accounts.
+
+## Abstract
+
+This EIP proposes to add a gas penalty for accesses to the account trie, where the address being looked up does not exist. Non-existing accounts can be used in 
+DoS attacks, since they bypass cache mechanisms, thus creating a large discrepancy between 'normal' mode of execution and 'worst-case' execution of an opcode. 
```

Comment:
A lot of this feels like motivation, not abstract.  Abstract should be a terse human-friendly version of the specification section.